cuebot/scheduler-sim/metrics.py

```
@@ -0,0 +1,104 @@
+import farm_spec
+
+PSQL = farm_spec.psql_cmd()
+FARM_CP = farm_spec.total_cores() * 100  # honor SIM_HOST_COUNTS small-farm mode
+FARM_HOSTS = farm_spec.total_hosts()     # ditto for the host-count denominator
+
+DURATION = int(sys.argv[1]) if len(sys.argv) > 1 else 75
+STEP = 2.0
+
+
+def q(sql):
+    out = subprocess.run(PSQL + ["-c", sql], capture_output=True, text=True).stdout
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)